projects/desktop/src/assets/locales/ES.json

```diff
@@ -44,6 +44,10 @@
         }
       },
 
+      "onboarding": {
+        "title": "¡Ya casi estás <span>ahí</span>!"
+      },

```

Comment:
Why there is not this part:
"form": {
"nickname": "Pseudo",
"birthdate": "Date de naissance",
"finish": "C'est parti !"
}